crates/typst/src/math/equation.rs

```diff
@@ -177,6 +190,7 @@ impl Show for Packed<EquationElem> {
 impl ShowSet for Packed<EquationElem> {
     fn show_set(&self, styles: StyleChain) -> Styles {
         let mut out = Styles::new();
+        out.set(EquationElem::set_column_gap(self.column_gap(styles)));
```

Comment:
Shouldn't this be inside the first if-branch below since, I assume, the column gap will only apply to block level equations?

Reply:
I guess it doesn't really matter, but yeah probably. I've moved it into the first if-branch.